src/careamics_restoration/engine.py

```diff
@@ -532,12 +532,21 @@ def save_as_bioimage(
         if model_specs is not None:
             specs.update(model_specs)
 
+        # set in/out axes from config
```

Comment:
mean/std also need to be updated. I couldn't find it here.

Reply:
Oh, that I forgot. The mean/std in pre/post processing should be replaced by mean/std from the config?

Reply:
Yes :)

Reply:
Just like in https://github.com/juglab/napari-n2v/blob/40317fefbb41e0b44df354440c61616b8d13bbf1/src/napari_n2v/utils/io_utils.py#L245

Reply:
Done!
To pass the test, I let the mean & std have some values in default specs, then if we have a trained model `config.data.mean` should have some value and so the replacement happens.